UNFCCC_GHG_data/UNFCCC_reader/Guinea/read_GIN_BUR1_from_pdf.py

```diff
@@ -0,0 +1,905 @@
+                                                        unit_row=inv_conf["unit_row"],
+                                                        entity_row=inv_conf["entity_row"],
+                                                        regexp_entity=".*",
+                                                        regexp_unit=".*",
+                                                        default_unit="Gg")
+
+    print("Added unit information.")
+    # set index
+    df_energy_year = df_energy_year.set_index(inv_conf["index_cols"])
+
+    # convert to long format
+    df_energy_year_long = pm2.pm2io.nir_convert_df_to_long(df_energy_year, inv_conf["year"][page],
+                                                           inv_conf["header_long"])
+
+    # extract from tuple
+    df_energy_year_long["orig_cat_name"] = df_energy_year_long["orig_cat_name"].str[0]
+
+    # prep for conversion to PM2 IF and native format
+    # make a copy of the categories row
+    df_energy_year_long["category"] = df_energy_year_long["orig_cat_name"]
+
+    # replace cat names by codes in col "category"
+    # first the manual replacements
+    df_energy_year_long["category"] = df_energy_year_long["category"].str.replace('\n', '')
+    df_energy_year_long["category"] = \
+        df_energy_year_long["category"].replace(inv_conf["cat_codes_manual"]['energy'])
+
+    df_energy_year_long["category"] = df_energy_year_long["category"].str.replace(".", "")
+
+    # then the regex replacements
+    repl = lambda m : m.group('code')
+    df_energy_year_long["category"] = \
+        df_energy_year_long["category"].str.replace(inv_conf["cat_code_regexp"], repl,
+                                                    regex=True)
+
+    df_energy_year_long = df_energy_year_long.reset_index(drop=True)
+
+    df_energy_year_long["data"] = df_energy_year_long["data"].str.replace(",", ".")
+    df_energy_year_long["data"] = df_energy_year_long["data"].str.replace("NE1", "NE")
+
+    # make sure all col headers are str
+    df_energy_year_long.columns = df_energy_year_long.columns.map(str)
+    df_energy_year_long = df_energy_year_long.drop(columns=["orig_cat_name"])
+
+    df_energy_dict[page] = df_energy_year_long
+
+# combine all tables
+df_energy = pd.concat([df_energy_dict['116'], df_energy_dict['117'], df_energy_dict['118'], df_energy_dict['119']],
+                      axis=0,
+                      join='outer').reset_index(drop=True)
+
+print("Converting to interchange format.")
+df_energy_IF = pm2.pm2io.convert_long_dataframe_if(
+    df_energy,
+    coords_cols=coords_cols,
+    coords_defaults=coords_defaults,
+    coords_terminologies=coords_terminologies,
+    coords_value_mapping=coords_value_mapping['energy'],
+    filter_remove=filter_remove,
+    meta_data=meta_data,
+    convert_str=True,
+    time_format="%Y",
+)
+
+### convert to primap2 format ###
+data_pm2_energy = pm2.pm2io.from_interchange_format(df_energy_IF)
+
+
+# ###
+# 3. Read in LULUCF table
```

Comment:
It's AFOLU (Agriculture, Forestry and Land Use) and not LULUCF (Land Use, Land Use Change, and Forestry)
```suggestion
# 3. Read in AFOLU table
```

Reply:
Alright. Changed it everywhere in the script.